flags/parse.go

```diff
@@ -0,0 +1,89 @@
+package flags
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+type flagFormat uint8
+
+const (
+	flagFormatEnv    = flagFormat(1)
+	flagFormatEnvB64 = flagFormat(1 << 1)
+	flagFormatB64    = flagFormat(1 << 2)
+	flagFormatFile   = flagFormat(1 << 3)
+)
+
+var (
+	ErrEnvironmentVariableNotFound = fmt.Errorf("environment variable not found")
+)
+
+func fromEnv(v string) (string, error) {
+	result := os.Getenv(v)
+	if result == "" {
+		return "", ErrEnvironmentVariableNotFound
+	}
+
+	return result, nil
+}
+
+func fromBase64(v string) (string, error) {
+	return decode64(v)
+}
+
+func fromFile(v string) (string, error) {
+	resultBytes, err := readFromFile(v, true)
+	// TODO is the special EOF handling needed?
```

Comment:
```suggestion
```
Not sure what this about  anymore but I assume it would have come up in a test if it were an issue

Reply:
I don't think it is needed. I removed it. readFromFile uses os.ReadFile which does not return EOF.